HW0/P6/P6B.py

```diff
@@ -16,16 +16,26 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(-i) for i in xrange(7)]
```

Comment:
You also might want to look into np.logspace() it essentially does this list comprehension but with the magic of numpy!